From a21652bfb50b240936124de0dbc288418106e5de Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 12 May 2021 16:47:05 +0100
Subject: [PATCH] dtoverlays: Update /__symbols__/i2c0 from i2c0
 overlay

Update the symbol as well as the alias so that other overlays will
then be applied against the right node.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c0-overlay.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm/boot/dts/overlays/i2c0-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c0-overlay.dts
@@ -65,6 +65,14 @@
 			i2c0 = "/soc/i2c@7e205000";
 		};
 	};
+
+	fragment@8 {
+		target-path = "/__symbols__";
+		__overlay__ {
+			i2c0 = "/soc/i2c@7e205000";
+		};
+	};
+
 	__overrides__ {
 		pins_0_1   = <0>,"+1-2-3-4";
 		pins_28_29 = <0>,"-1+2-3-4";
